### DISCUSSION TITLE
Allow multiple ClockBoards for multi-centre cities (for statistical modelling)

### DISCUSSION THREAD

**Discussion by mtennekes** [zonebuilders/zonebuilder#30] [Ideas]
One remaining question is what to do with cities that have multiple clear city centres. Merging two partly overlapping ClockBoards leads to complex zone shapes.

However, why should we have to merge the two ClockBoards? Do we really need to have one zoning system for each city? 

My take is that two zoning systems (one for each centre) can perfectly co-exist, especially for statistical modelling. Consider a ClockBoard as a 2d kernel density of 'city buzz' for a city with one centre. Then for a city with two (or more) centres, the corresponding ClockBoards represent partly overlapping 2d kernel densities. This makes sense in daily life: probably the area between the two centres is extremely popular.

Taking even a step further, we can model a large city with one centre with one ClockBoard, and add small-scale clockboards for each neighbourhood that has a clear centre (e.g. local market or transport hub). For daily use of the ClockBoard system, this is probably too complex, but for statistical modelling it can be useful I think.

Please share your opinions on this matter.

**Comment by Robinlovelace** [comment]
I think that's a good idea @mtennekes but would be worth seeing it in practice. I think it would be useful for showing situations when the sphere of influence of 2 or more cities interact but I do think selecting the size of the outer ring, relating to the size of the sphere of influence, is important.
